test/integration/main_test.go

```diff
@@ -375,27 +396,41 @@ func queryPmapEvents(ctx context.Context, query *bigquery.Query) ([]*v1alpha1.Pm
 	if err != nil {
 		return nil, retry.RetryableError(fmt.Errorf("failed to read job: %w", err))
 	}
-	var pmapEvents []*v1alpha1.PmapEvent
+
+	var entries []*bqEntry
 	for {
-		var row []bigquery.Value
-		err := it.Next(&row)
+		var entry []bigquery.Value
```

Comment:
Just curious, can BQ automatically handle the unmarshal when we use `var entry bqEntry` here?

Reply:
I tried, it can't, that's why I did it this way.